### DISCUSSION TITLE
[Meta] Maintaining the project

### DISCUSSION THREAD

**Discussion by TLouf** [academicpages/academicpages.github.io#1982] [General]
Hi,

First I'd like to thank everyone who contributed so far to @academicpages (and especially @staeiou for initiating it) which has certainly, as it has for me, saved a lot of time for researchers around the world who try to publicise their work. However, despite its popularity, proven by the frequency of issue opening or the sheer number of forks, no commit has been made since more than two years ago. It seems that @staeiou is the only collaborator able to merge PRs on the repo, and it is normal that with time, one ends up contributing less time to such a project for all sorts of reasons. Hence the current situation. Naturally then, new collaborators should come in to keep it alive. I have no idea how new collaborators could be selected, and it is up to @staeiou I guess, but there are many small things one can do with minimal experience.

#545 for instance is a trivial PR to merge, and there are more. Most issues can be answered and closed extremely simply by anyone who made their own page. I personally have ideas I've implemented on my personal page for which I could write PRs, but with no hope of a merge these kinds of initiatives are killed in the egg. So whether it is implementing new ideas or even changes made upstream in Minimal Mistakes (like the search), there are a lot of things that could be done, and, I'm sure, the people willing to contribute. I myself have no legitimacy in this regard because I'm no web developer, but I'd be glad to help!

Cheers!

**Comment by armack** [comment]
I very much agree with this, as it seems there are several little issues/annoyances that people have been fixing for a couple of years which are sitting as pull requests. There's also possibility to port in changes from Minimal Mistakes as it continues to have active development and some of those changes could possibly be useful.

I do want to point out though, that the change in #545 would seemingly break multi-author support on the publications page. And while that would be fine for probably 99.9+% of people using the theme, it seems silly to break an existing feature when there's an equally trivial copy/paste that fixes it without breaking anything which I just requested in #566.

**Comment by rjzupkoii** [comment]
@staeiou has stepped back from active maintenance of the project and has handed things off to me. I'll be working the next couple days to clear out the backlog and setup a process for pull requests. In the meantime, please reach out if you might also be interested in helping to maintain the project!
